src/containers/ScreenContainer.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
import React, { Component } from 'react';
```